ingest/src/test/java/com/microsoft/azure/kusto/ingest/ResourceManagerTest.java

```diff
@@ -122,7 +122,7 @@ private static Results generateIngestionResourcesResult() {
         valuesList.add(new ArrayList<>((Arrays.asList("TempStorage", STORAGE_2))));
         valuesList.add(new ArrayList<>((Arrays.asList("IngestionsStatusTable", STATUS_TABLE))));
 
-        return new Results(colNameToIndexMap, colNameToTypeMap, valuesList);
+        return new Results(colNameToIndexMap, colNameToTypeMap, valuesList, "");
```

Comment:
shouldn't this be an overload?

Reply:
instead of passing an empty string , shouldn't we add another overload without the extra param?